bin/recog_verify

```diff
@@ -35,6 +38,10 @@ option_parser = OptionParser.new do |opts|
     options.warnings = o
   end
 
+  opts.on("--schema-location SCHEMA_FILE", "Location of the Recog XSD schema file (default: ./xml/fingerprints.xsd). If not specified, validation will not be run.") do |schema_file|
```

Comment:
Should we mention "default: ./xml/fingerprints.xsd" in the description if that isn't used by default?

Reply:
Ah yeah. Initially I had it force schema checking by default, but decided to disable that (partly because I didn't want to assume where fingerprints.xsd would be). I'll update the documentation. Good catch.